app/controllers/content_controller.rb

```diff
@@ -2,11 +2,7 @@ class ContentController < Api::BaseController
   before_action :validate_params!
 
   def show
-    @content = Queries::FindContent.retrieve(
-      from: from,
-      to: to,
-      organisation_id: params[:organisation_id]
-    )
+    @content = Queries::FindContent.retrieve(content_filters: api_request)
```

Comment:
I still feel this function arguments are still hidden. It's not clear what parts of an "API request" this function will use or what will be used as filters.